src/Services/Bookings/Booking.API/Controllers/BookingController.cs

```diff
@@ -36,33 +37,30 @@ public async Task<IActionResult> GetBookingsAsync()
         public async Task<IActionResult> GetBookingsByUserIdAsync(string data)
         {
             Guid userId = JsonConvert.DeserializeObject<Guid>(data);
-            return Ok(await _bookingRepository.GetBookingsByUserAsync(userId));
+            return Ok(await _bookingLogic.GetBookingsByUserIdAsync(userId));
         }
 
         [HttpPost]
         [Route("AddBooking")]
         public async Task<IActionResult> AddBookingAsync([FromBody] Booking data)
         {
-            await _bookingRepository.AddBookingAsync(data);
-
+            await _bookingLogic.AddBookingAsync(data);
             return Ok(true);
```

Comment:
It does not make sense to return bool value. It is enough to return empty result "return Ok();".
On client you will check which HTTP code was returned. If it was 200, it means everything is correct.